google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/BaseTest.java

```diff
@@ -26,4 +31,26 @@
   public static String generateRandomName(String placeholder) {
     return "gapic-" + placeholder + UUID.randomUUID().toString().substring(0, 8);
   }
+
+  public void waitGlobalOperation(Operation operation) throws IOException {
+    GlobalOperationsClient operationsClient = GlobalOperationsClient.create();
+    long startTime = System.currentTimeMillis();
+    while (true) {
```

Comment:
Please update all wait functions to include some `Thread.sleep()` calls, otherwise they hit the server too frequently (here and in the other already existing wait() functions you already have). Please do the similar thing in the other languages if you don't do it already. 

In general, even for simplest polling strategies (and we definitely do not need anything more sophisticated for this case) we always need to add some delay between the calls, otherwise it becomes extremely cons-ineffective for both client and the server. Specifically we may end up making hundreds or even thousands of requests before getting a positive response, which puts great load on the server, client and the network, but lets to save usually only second or two compared to polling with a delay (which itself leads in order of magnitude lower number of polling requests).